src/libraries/System.Private.CoreLib/src/System/IO/Path.Unix.cs

```diff
@@ -13,6 +13,12 @@ public static partial class Path
 
         public static char[] GetInvalidPathChars() => new char[] { '\0' };
 
+        // Checks if the given path is available for use.
+        private static bool ExistsCore(string fullPath)
+        {
+            return Interop.Sys.LStat(fullPath, out Interop.Sys.FileStatus _) == Interop.Errors.ERROR_SUCCESS;
```

Comment:
I am not sure if `ERROR_SUCCESS` is defined in the code for Unix.

```suggestion
            return Interop.Sys.LStat(fullPath, out Interop.Sys.FileStatus _) == Interop.Errors.SUCCESS;
```

Reply:
I can't seem to find Interop.Errors.SUCCESS on Mac OS, 
However the Interior.Errors.ERROR_SUCCESS is resolving to 0 on Mac OS,
 so I assume it'd be same with Linux as well